lib/run-context.js

```diff
@@ -147,6 +147,28 @@ RunContext.prototype.inDir = function (dirPath, cb) {
 };
 
 /**
+ * Change directory into provided directory.
+ * @param  {String} dirPath - Directory path (relative to CWD). Prefer passing an absolute
+ *                            file path for predictable results
+ * @param {Function} [cb] - callback who'll receive the folder path as argument
+ * @return {this} run context instance
+ */
+
+RunContext.prototype.inDirKeep = function (dirPath, cb) {
```

Comment:
The name of this function is not really descriptive IMO. Maybe we should rename it `.cd(dirPath)`? Or something else that makes it clear it only will `cd` the active process?